code/datums/components/storage/concrete/pockets.dm

```diff
@@ -34,6 +34,7 @@
 
 /datum/component/storage/concrete/pockets/shoes/Initialize()
 	. = ..()
+	cant_hold = typecacheof(list(/obj/item/screwdriver/power))

```

Comment:
```suggestion
	cant_hold = typecacheof(list(/obj/item/screwdriver/power)) //Must be specifically called out since normal screwdrivers can fit but not the wrench form of the drill
```